starknet_devnet/blueprints/feeder_gateway.py

```diff
@@ -162,6 +162,8 @@ def get_transaction_status():
 
     transaction_hash = request.args.get("transactionHash")
     transaction_status = state.starknet_wrapper.transactions.get_transaction_status(transaction_hash)
+    if "block_hash" in transaction_status:
```

Comment:
Shouldn't this be handled in `DevnetTransactions.get_transaction_status`? There we already have one `if` which checks if `"block_hash"` exists.

Reply:
Whatever I touch it opens a new construction site 👷 , so I wanted to avoid it, but you are right, will fix it the proper way.

Reply:
b58288c9f2b2959f516250c2ae767ee8d989706a